library/src/app/components/trade-confirm/trade-confirm.component.ts

```diff
@@ -100,9 +100,7 @@ export class TradeConfirmComponent implements OnInit, OnDestroy {
           const message = this.translatePipe.transform(
             'trade.confirm.error.quote'
           );
-          this.snackBar.open(message, undefined, {
-            duration: 3000
-          });
+          this.snackBar.open(message, 'OK');
```

Comment:
Is "OK" here a string that's displayed?

Reply:
Yes, just to ensure that the error is acknowledged.